source/extensions/filters/http/transformation/inja_transformer.h

```diff
@@ -154,6 +154,7 @@ class InjaTransformer : public Transformer {
 
   absl::optional<inja::Template> body_template_;
   bool merged_extractors_to_body_{};
+  std::vector<std::tuple<std::string, bool, inja::Template>> merge_templates_;
```

Comment:
nit: can you add a comment about what these fields are so when cruising through the `.h` it's more clear?